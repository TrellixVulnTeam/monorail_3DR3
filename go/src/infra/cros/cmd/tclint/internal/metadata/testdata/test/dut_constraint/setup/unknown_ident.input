{
    "remoteTestDrivers": [
        {
            "name": "remoteTestDrivers/tauto",
            "tests": [
                {
                    "name": "remoteTestDrivers/tauto/tests/foo",
                    "dut_constraint": {
                        "setup": {
                            "expression": "duts.setup.peripheral.servo.present"
                        }
                    }
                }
            ]
        }
    ]
}