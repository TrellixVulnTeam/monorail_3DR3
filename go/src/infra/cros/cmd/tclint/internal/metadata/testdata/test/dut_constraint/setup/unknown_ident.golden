[
"## Errors ##",
"Test 'remoteTestDrivers/tauto/tests/foo': dut_constraint: setup: compile expression: ",
"  ERROR: \u003cinput\u003e:1:1: undeclared reference to 'duts' (in container 'chromiumos.config.api.test.dut.v1')",
"   | duts.setup.peripheral.servo.present",
"   | ^"
]